IPs/RSKIP143.md

```diff
@@ -0,0 +1,42 @@
+# Segwit Compatibility
+
+|RSKIP          |143          |
+| :------------ |:-------------|
+|**Title**      |Segwit Compatibility|
+|**Created**    |22-OCT-19 |
+|**Author**     |PGP |
+|**Purpose**    |Sca,USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft|
+
+## Abstract
+
+This document describes the procedure to allow the 2wp proccess to accept segwit transactions sent using a P2SH-PWPKH wallet. 
+
+## Motivation
+
+The 2wp proccess only allows transactions has been sent using a P2PKH address. This RSKIP describes how Bridge could accept P2SH-PWPKH[1] since this is one of the most popular wallet nowadays and the segwit transactions uses less space in a block. 
```

Comment:
```suggestion
The 2wp proccess only accepts transactions that were sent using a P2PKH address. This RSKIP describes how Bridge could accept P2SH-PWPKH[1] addreses since this is one of the most popular wallet nowadays and the segwit transactions use less space in a block. 
```